### PR TITLE
Traviscli

### PR DESCRIPTION
**Pull request by thewchan**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by thewchan** [comment]
@conda-forge-admin, please restart ci

**Comment by thewchan** [comment]
@conda-forge-admin, please restart ci

**Comment by thewchan** [comment]
@conda-forge/staged-recipes ready for review!

File: recipes/traviscli/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  run:
+    - click
+    - python >=3.6
+    - pytravisci
+
+test:
+  imports:
+    - travisci
+    - travisci.exceptions
+  commands:
```
Comment (synapticarbors) [inline recipes/traviscli/meta.yaml:33]: Can you also check that the entry point is working:

```
commands:
   - pip check
   - traviscli --help
```

Comment (thewchan) [reply recipes/traviscli/meta.yaml:33]: yup I'll fix this on the feedstock

Comment (synapticarbors) [reply recipes/traviscli/meta.yaml:33]: Any reason not to just include it here before the merge?

Comment (thewchan) [reply recipes/traviscli/meta.yaml:33]: Argh because I had a brainfart and I saw that close better and thought this was already merged. I can fix this now

**Pull request merged**